src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Product/Combination/paginated_list.html.twig

```diff
@@ -30,6 +30,7 @@
       'productId': productId,
       'isMultistoreActive': isMultistoreActive,
       'contextShopName': contextShopName,
+      'contextShopId': contextShopId,
```

Comment:
Here my comment about using `context` in the naming makes less sense, but if we don't use it in the data attributes for row buttons I guess it would be more consistent not to use it here either So I guess contextShopName and contextShopId should be replaced with shopName and shopId